File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
C = int(input())

for _ in range(C):
    scores = list(map(int, input().split()))
    sum = 0
    for i in range(1, len(scores)):
        sum += scores[i]
    average = sum / scores[0]
    count = 0
    for i in range(1, len(scores)):
        if average < scores[i]:
            count += 1
    print(f"{(count * 100 / scores[0]):.3f}%")

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int C = sc.nextInt();

        for (int i = 0; i < C; i++) {
            int num = sc.nextInt();
            int[] scores = new int[num];

            int sum = 0;
            for (int j = 0; j < num; j++) {
                scores[j] = sc.nextInt();
                sum += scores[j];
            }

            double average = (double) sum / num;
            int count = 0;

            for (int j = 0; j < num; j++) {
                if (scores[j] > average) count++;
            }

            double rate = ((double) count / num) * 100;
            System.out.printf("%.3f%%\n", rate);
        }

        sc.close();
    }
}